internal/app/secretless/listeners/http/listener.go

```diff
@@ -226,5 +223,13 @@ func (l *Listener) GetName() string {
 
 // ListenerFactory returns a Listener created from options
 func ListenerFactory(options plugin_v1.ListenerOptions) plugin_v1.Listener {
-	return &Listener{BaseListener: plugin_v1.NewBaseListener(options)}
+	httpConfig, err := v2.NewHTTPConfig(options.ServiceConfig.ConnectorConfig)
```

Comment:
We still need to figure this out but ideally the logic for processing configBytes should reside inside the connector itself. However, we'll need to have the connectors carry out validation at config construction so that we fail fast.

Reply:
That's a interesting point.  I was assuming they would fail when _they_ were constructed, which would also happen Secretless boots up, so from a UX perspective that's still failing fast.  Otherwise we'd have to add to the Connector API, and that would get even more complicated over gRPC, etc.  I think it could get messy.